src/__tests__/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from '../components/App';
+
+import Enzyme,  {shallow}  from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+Enzyme.configure({ adapter: new Adapter() });
+
+it('renders without crashing', () => {
+  let mountedApp = shallow(<App />)
```

Comment:
@ruganda, another missing semi-colon.

Also, the name of this test is misleading since it doesn't actually use `ReactDOM` to render anything. Maybe you could consider rendering `<App />` or just changing the name of the test.

Reply:
I think the name of the test is descriptive enough. we are using `shallow` rendering from `enzyme`

Reply:
Oh, ok. Thanks for the clarification.
but the semi-colon is still missing